internal/grpcwrap/provider.go

```diff
@@ -60,13 +63,13 @@ func (p *provider) GetSchema(_ context.Context, req *tfplugin5.GetProviderSchema
 
 	for typ, res := range p.schema.ResourceTypes {
 		resp.ResourceSchemas[typ] = &tfplugin5.Schema{
-			Version: res.Version,
+			Version: int64(res.Version),
```

Comment:
The type here shouldn't have changed, does this need a conversion?

Reply:
Sorry, this is a remnant of the `SchemaForResourceType` changes. Nice catch!